core/java/src/org/openda/exchange/timeseries/TimeSeries.java

```diff
@@ -1065,9 +1064,8 @@ public double getInterpolatedExtraValueAsDouble(String key, double time) {
 	 * Show Type of this object for proper casting
 	 */
 	
-	@SuppressWarnings("rawtypes")
 	public Class getValueType() {
-		return org.openda.utils.Array.class;
+		return org.openda.exchange.timeseries.TimeSeries.class;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.openda.exchange.timeseries.TimeSeries' due to existing same package import 'org.openda.exchange.timeseries.*'](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5102854)